l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1038,6 +1038,14 @@ def strip_xml_content(self, xml):
                 elem.text = elem.text.strip()
         return etree.tostring(root)
 
+    def remove_x509_sign(self, xml):
```

Comment:
in che senso? e' un certificato ma la frima e' XAdES e quindi rimuovo i tag non riconosciuti da pyxB